src/olympia/devhub/templates/devhub/addons/submit/upload.html

```diff
@@ -31,6 +31,9 @@ <h3>{{ _('Upload Version') }}</h3>
     Use the fields below to upload your add-on package. After upload, a series
     of automated validation tests will be run on your file.
   {% endtrans %}
+  {% trans %}
+    The maximum file size accepted is 200 MB. If your add-on is larger than 200 MB, it will fail validation.
```

Comment:
Why the extra explanation here and not in the other place?

Reply:
Just going off the linked issue 😅